### PR TITLE
[Config] Update test suites for CephAdm permissive mode scenarios

### PR DESCRIPTION
**Pull request by vamahaja**
Fix consists of -

- Rename test suite file name
- Add test suites to metadata pipeline

### CODE REVIEW

**Review by pranavprakash20 — APPROVED** [review]
LGTM

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -182,6 +182,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
           metadata:
             - rados
+        - name: "Permissive Mode Scenarios"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/5.3.yaml:186]: Please add time taken for suite execution.

File: pipeline/metadata/6.0.yaml [hunk 2]
```diff
@@ -226,6 +226,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rados
+        - name: "Permissive Mode Scenarios"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/6.0.yaml:230]: Please add time taken for suite execution.

File: pipeline/metadata/6.1.yaml [hunk 3]
```diff
@@ -222,6 +222,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rados
+        - name: "Permissive Mode Scenarios"
+          execution_time: ""
```
Comment (manasagowri) [inline pipeline/metadata/6.1.yaml:226]: Please add time taken for suite execution.

**Review by manasagowri — COMMENTED** [review]
Just a few small changes requested.

**Review by tintumathew10 — APPROVED** [review]
Looks good to me

**Pull request merged**